lib/models/addon.js

```diff
@@ -638,11 +638,7 @@ var Addon = CoreObject.extend({
       return tree;
     }
 
-    var processed = this.preprocessJs(tree, '/', this.name, {
```

Comment:
We should leave the `preprocessJs` call in by default. This is definitely the path we want to take.